src/frontend/next/next.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  distDir: 'build',
```

Comment:
Why do we have to specify a build dir at all?

Reply:
If not, outputs to `out` which doesn't follow our previous Gatsby semantics. I switched to `build` so that it's following similar semantics.